[Docker] Tensorflow conflict

# Distribution
`ocrd/all:maximum-git` 
Id: sha256:f4a95c7ce9533446f91de9bd076911e2ebbad4e265fa97bc05e5b84f7ccaa2aa
Created: 2020-06-17T13:50:54.535025462Z

# Plattform
Ubuntu 18.04 LTS

# Log

```
2020-06-18 10:21:01,534.534 INFO ocrd.workspace - Saving mets '/data/1000/mets.xml'
building OCR-D-SEG-PAGE-anyocr from OCR-D-BINPAGE-sauvola with pattern rule for ocrd-anybaseocr-crop
STAMP=`test -e OCR-D-SEG-PAGE-anyocr && date -Ins -r OCR-D-SEG-PAGE-anyocr`; ocrd-anybaseocr-crop   -I OCR-D-BINPAGE-sauvola -p OCR-D-SEG-PAGE-anyocr.json -O OCR-D-SEG-PAGE-anyocr --overwrite 2>&1 | tee OCR-D-SEG-PAGE-anyocr.log && touch -c OCR-D-SEG-PAGE-anyocr || { if test -z "$STAMP"; then rm -fr OCR-D-SEG-PAGE-anyocr; else touch -c -d "$STAMP" OCR-D-SEG-PAGE-anyocr; fi; false; }
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (tensorflow 1.15.3 (/usr/lib/python3.6/site-packages), Requirement.parse('tensorflow>=2.0'), {'ocrd-anybaseocr'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/ocrd-anybaseocr-crop", line 33, in <module>
    sys.exit(load_entry_point('ocrd-anybaseocr', 'console_scripts', 'ocrd-anybaseocr-crop')())
  File "/usr/bin/ocrd-anybaseocr-crop", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.6/site-packages/importlib_metadata/__init__.py", line 96, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/build/ocrd_anybaseocr/ocrd_anybaseocr/cli/cli.py", line 3, in <module>
    from ocrd.decorators import ocrd_cli_options, ocrd_cli_wrap_processor
  File "/build/core/ocrd/ocrd/__init__.py", line 17, in <module>
    from ocrd.processor.base import run_processor, run_cli, Processor
  File "/build/core/ocrd/ocrd/processor/__init__.py", line 1, in <module>
    from .base import (
  File "/build/core/ocrd/ocrd/processor/base.py", line 6, in <module>
    from ocrd_utils import getLogger, VERSION as OCRD_VERSION
  File "/build/core/ocrd_utils/ocrd_utils/__init__.py", line 135, in <module>
    from .logging import * # pylint: disable=wildcard-import
  File "/build/core/ocrd_utils/ocrd_utils/logging.py", line 22, in <module>
    from .constants import LOG_FORMAT, LOG_TIMEFMT
  File "/build/core/ocrd_utils/ocrd_utils/constants.py", line 4, in <module>
    from pkg_resources import get_distribution
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3260, in <module>
    @_call_aside
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3244, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3273, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'tensorflow>=2.0' distribution was not found and is required by ocrd-anybaseocr
Makefile:313: recipe for target 'OCR-D-SEG-PAGE-anyocr' failed
make[1]: *** [OCR-D-SEG-PAGE-anyocr] Error 1
make[1]: Leaving directory '/data/1000'
make: *** [1000] Error 2
Makefile:204: recipe for target '1000' failed
make: Leaving directory '/data'
```

--- a/Makefile
+++ b/Makefile
@@ -406,8 +406,8 @@ $(OCRD_EXECUTABLES): | $(BIN)/wheel
 fix-pip:
 	. $(ACTIVATE_VENV) && $(PIP) install --force-reinstall $(PIP_OPTIONS_E) \
 		opencv-python-headless \
-		"pillow>=6.2.0" \
-		$$($(PIP) list | grep tensorflow-gpu | sed -E 's/-gpu +/==/')
+		"pillow>=7.1.0" \
+		$$($(PIP) list | grep tensorflow | sed -nE '/tensorflow +1[.]/s/ +/>=2.0/p;/tensorflow-gpu +2[.]/s/ +/~=1.15.3/p')
 
 # At last, we know what all OCRD_EXECUTABLES are:
 all: $(OCRD_EXECUTABLES)

